Resources/Styles/package_list.scss

```diff
@@ -22,9 +22,9 @@
   li {
     background-color: var(--package-list-background);
     border-left: 3px var(--package-list-border) solid;
+    display: grid;
 
     a {
-      display: block;
       padding: 10px;
```

Comment:
I've added an `align-items: center;` here to bring the display of the star information to be vertically centered.